### ISSUE TITLE
Command and Security Antagonist Chances

### ISSUE THREAD

**Issue by bathosbathtime** [d3athrow/vgstation13#25964]
[discussion] [featurerequest] 

#### Description of issue
For a long time, security and implanted heads' antagonists chances were bugged, meaning they would not roll at all. This has somewhat recently been fixed.
I feel that there has been enough time of antagonist rolls for these roles for the community to get a good idea of if they want them. An official channel to talk about if they want them in or want them gone seems appropriate. The thread has discussed this lightly but the thread is also not the best place for getting attention from our coding team, nor is it particularly constructive anyway.

The benefits are pretty clear, in that the command and security roles being even more trustworthy will give them 'meta' trustworthiness. While I don't like it, we can not ask our players to simply trust someone they know they can't due to out of character reasons.
There are also some clear downsides. This will make the game more game-like with 'confirmed' roles, which may not be productive towards a roleplaying environment. There is also the fact that this may simply be an attempt to make roles that require charisma to pull off have more 'charisma' through artificial means, avoiding the actual issue of the players playing the roles. There is also the fact that implanted roles already have a reduced chance, but I find that that does not matter much if the playerbase knows the roles can be antagonists at all.

How this will be done and which roles to include are also in question. While it makes sense in character to use all loyalty implanted roles, the non-implanted heads being jokes could also stand to be addressed. I hope we can find a solution that we all can agree on, at least mostly.

Thank you for taking the time to read this.

EDIT: To be clear, the requested feature is having implanted roles have zero chances for antagonist. Though this issue more exists to see if anyone else wants that, or if they want something similar, or if there isn't any support for this sort of thing at all.

**Comment by Accipitermaris** [comment]
TL;DR: **I think we just need a clearer ruling on [Doing The Right Thing] vs. [Following the Chain of Command]. Also, Space Law takes precedence over taking orders (you're not supposed to follow illegal orders from a boss), but this isn't always recognized by admins.**

The problem is basically this: as a Security Officer, if your HoS is executing innocents, with even the thinnest veil of probable deniability, you are 100% valid to unquestioningly support that HoS. But the inverse ain't true. If you arrest a superior that has very obviously broken Space Law, you run into all sorts of problems because Space Law itself is basically written with the idea that you always bow to your superiors, or at least with the idea that a reliable superior always exists, when this is not always the case.

Good example of this is that round when the HoP and sole head of staff was randomly giving out all access to people, sec saw that as abuse of power, they arrested him, but there was no one on the chain of succession to replace him so they basically just coup'd themselves into ultimate authority. You ask a bunch of admins on that situation and they're all gonna have different viewpoints.

**This creates serious bad juju in the sense in that you're in less danger of getting bwoinked by supporting an obvious antagonist head of staff as Sec than you are in following Space Law and arresting him.**

**Comment by Blind-Watcher** [comment]
Proposed solution: Create suggested courses of action in Space Law.
"So you're an inferior non-head who suspects your superior head is doing something illegal? What do you do?" and then it goes on to suggest steps like going to the IAA, informing another head, and so forth. At some point, mutiny should be ostensibly permissible by Centcomm; but this shouldn't mean that silentshuffle stunarrest the minute a sec officer disagrees with the HOS is permitted. I know this sounds like common sense, but it really should be written out in Space Law. At the very least to use as precedent for an officer defending himself against mutiny accusations at trial.

**Comment by SonixApache** [comment]
make following space law the norm not the exception - i've been bitching about this for months

anyone should be able to be a traitor, it helps create the unease of not being able to wholly trust your co-workers which i feel is important in our game of spess

avoid metagay by avoiding metachecks "sec can't be antag" - allows for more dynamic, fun and arrpee

**Comment by ShiftyRail** [comment]
>For a long time, security and implanted heads' antagonists chances were bugged, meaning they would not roll at all.

This isn't the case. I've seen plenty of traitor caps, traitor officers, vampsec, and lingsec prior to antag datums.
If anything, there was no reduced chance and sec/command was normal as far as antagonist chances were considered.
If you want examples, you can think of Archinist (rip), or Horatio as a traitor captain.

What Accip says is correct, and the admin "double-standard" could be an issue here ; however to me the solution isn't to make space law more powerful or more detailled. I think it works well as a baseline and framework open to interpretations.
It is also not that surprising that people prefer to support their department or the people they've been playing the round with. **Security isn't, by far, the department in which this happens the most,** just look at Cargo or Vox traders. Just remember that SS13 is a social game, with all the bias it implies.

I don't think the other two suggestions are going to work because of that ; one, "clan-like behaviour"  is a natural consequence of how the game is played, two, it's not unique to sec (at all)

My solution isn't probably the most popular one, but this all could be solved if Sec/Head of Staff traitors were encouraged, both ICly (with a special traitor text, and maybe a differently worded freeform objective for those who have freeform jecties?) and OOCly (already the case if you read the guide to antag sec and antag captain on the wiki) to use their job to be more dramatic and entertaining rather than focusing on depopulating the station.

Alternatively, admins could give a little help to the stuggling working class when they are oppressed by the yoke of antag sec (sending out NT inspectors to defuse the situation, giving advice in subtle PMs, etc...)
I don't think it would be that much of a big deal for admins ; they're not just the OOC police after all, they're supposed to be gamemaster and make the round more entertaining.
It also would fit with the "hands-off" approach that's preferred on /vg/, after all, the admins aren't intervening OOCly, just giving tools and helping the situation to be resolved ICly.

This could also apply to "organic" mutinies (ie without any antags on any side).

Of course, this raises the question of "antag standards/guidelines", but since the question is also about sec antagonism, to me it's not that out of context.

**Comment by yackemflam** [comment]
@Accipitermaris 
Technically the QM is supposed to replace the HoP if the HoP gets taken out and the Warden replace the HoS if the HoS gets taken out

Also, the game is supposed to have some gray areas that people WILL conflict over and WILL beat the shit out of each other for. So the only head HoP getting arrested and mutinied over should mean that the crew should figure out on what to do. Like appoint a new HoP by vote or whoever wants to be the new HoP or even use the HoP's id card to request a NEW HoP.
There are plenty of ways to do it.

But I also agree that Sec/Heads shouldn't get antag rolls as OFTEN as they are getting them, but for a different reason

**Comment by ShiftyRail** [comment]
They are 50% less likely to get traitor.
80% less likely to get vampire or changeling.

**Comment by west3436** [comment]
Dated discussion thread - most rulesets now have a list of protected jobs
